app/assets/javascripts/components/campaign/campaign_overview_handler.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import CampaignStatsDownloadModal from './campaign_stats_download_modal';
+
+const CampaignOverviewHandler = (props) => {
+  return <CampaignStatsDownloadModal {...props} />;
+};
+
+export default CampaignOverviewHandler;
```

Comment:
Is the purpose of this handler just to keep the general pattern?

Reply:
Yes. I figured something more general than just the stats modal is what made sense to put into the router, and this will make it inviting to start porting the rest of campaign overviews to React.